public/apps/configuration/constants.tsx

```diff
@@ -43,6 +43,9 @@ export const CLUSTER_PERMISSIONS: string[] = [
   'cluster:admin/opendistro/ad/detector/write',
   'cluster:admin/opendistro/ad/detectors/get',
   'cluster:admin/opendistro/ad/result/search',
+  'cluster:admin/opendistro/ad/tasks/search',
+  'cluster:admin/opendistro/ad/detector/preview',
```

Comment:
Thanks for the change. Just a minor nit: I believe the Security plugin team wants them to be sorted lexicographically.

Reply:
Got it, that will look better. Will change